st2actions/st2actions/scheduler/queuer.py

```diff
@@ -0,0 +1,76 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from kombu import Connection
+
+from st2common import log as logging
+from st2common.constants import action as action_constants
+from st2common.exceptions.db import StackStormDBObjectNotFoundError
+from st2common.models.db.liveaction import LiveActionDB
+from st2common.transport import consumers
+from st2common.transport import utils as transport_utils
+from st2common.transport.queues import ACTIONSCHEDULER_REQUEST_QUEUE
+from st2common.util import action_db as action_utils
+from st2common.util import execution_queue_db as exdb
+from st2common.services import action as action_service
+from st2common.persistence.execution_queue import ExecutionQueue
+
+__all__ = [
+    'ActionExecutionQueuer',
+    'get_queuer'
+]
+
+
+LOG = logging.getLogger(__name__)
+
+
+class ActionExecutionQueuer(consumers.MessageHandler):
+    message_type = LiveActionDB
```

Comment:
Let's please add a docstring which clarifies what that service / handler does.

I'm also still thinking about a better name - "queuer" is a bit vague and also hard to type.

Maybe "ActionExecutionDelayHandler" or something along those lines to make it more obvious what it's purpose is just by looking at the name.

Reply:
It's not a delay handler though. All executions funnel through it, delayed or not. Perhaps `ActionExecutionSchedulerEntrypoint` or something. I'm really struggling coming up with a better name. 😂